central/complianceoperator/v2/profiles/datastore/datastore_impl.go

```diff
@@ -22,6 +24,29 @@ type datastoreImpl struct {
 	db               pgPkg.DB
 	store            pgStore.Store
 	profileEdgeStore edge.Store
+	searcher         profileSearch.Searcher
+}
+
+// GetProfile returns the profile for the given profile ID
+func (d *datastoreImpl) GetProfile(ctx context.Context, profileID string) (*storage.ComplianceOperatorProfileV2, bool, error) {
+	if ok, err := complianceOperatorSAC.ReadAllowed(ctx); err != nil {
```

Comment:
Are we assuming that for SAC purposes that everyone has full access? I know that postgres will check some requirements of SAC, but I think this means that everyone has to have full access

Reply:
fair enough